src/HTTPure/Request.purs

```diff
@@ -51,12 +54,23 @@ fullPath request = "/" <> path <> questionMark <> queryParams
 -- | Given an HTTP `Request` object, this method will convert it to an HTTPure
 -- | `Request` object.
 fromHTTPRequest :: HTTP.Request -> Aff Request
-fromHTTPRequest request = pure
-  { method: Method.read request
-  , path: Path.read request
-  , query: Query.read request
-  , headers: Headers.read request
-  , body: Body.read request
-  , httpVersion: Version.read request
-  , url: requestURL request
-  }
+fromHTTPRequest request = do
```

Comment:
Following my prior comment, this will be simplified to:

```purs
fromHTTPRequest request = do
    body < liftEffect $ Body.read request
    pure { ... }
````

I believe you'll also be able to revert most of the import changes above too

Reply:
Updated the `Body.read` part. I think the import changes come from `npx purs-tidy format-in-place src test docs` command. Do you want me to revert those or run `purs-tidy` again at the end?

Reply:
Nah, that's fine, no worries!  Weird as I have purs-tidy in CI and it's never moved imports around for me before... but not a big deal to me at all